src/main/resources/de/qabel/desktop/ui/about/aboutPopup/aboutpopup.css

```diff
@@ -27,46 +27,65 @@
 }
 
 .text-area-popup {
-    -fx-max-width: 500px;
+    -fx-max-width: 405px;
     -fx-min-width: 100px;
-    -fx-max-height: 1000px;
-    -fx-min-height: 500px;
+    -fx-min-height: 400px;
 }
 
-.text-area-popup *.text {
+.align-text-center {
+    -fx-max-width: 405px;
+    -fx-overflow: scroll;
+    -fx-display: inline-block;
+    -fx-border-width: 0;
+    -fx-border-radius: 0;
+}
+
+.align-text-center * {
+     -fx-margin: 10px 5px 5px 5px;
+     -fx-padding: 10px 0px 0px 0px;
+}
+
+.align-text-center .text {
     -fx-text-alignment: center;
 }
 
 .text-area-popup * {
-    -fx-background-color: white;
-    -fx-overflow: scroll;
-    -fx-display: inline-block;
+    -fx-background-color: transparent;
     -fx-border-width: 0;
     -fx-border-radius: 0;
+    -fx-margin: 10px 5px 5px 5px;
     -fx-padding: 10px 0px 0px 0px;
+    -fx-overflow: scroll;
+    -fx-display: inline-block;
+}
+
+.text-area-popup .text {
+    -fx-text-alignment: LEFT;
 }
 
+.text-area-popup .content {
+    -fx-padding: 10px 10px 10px 10px;
+ }
+
 .fade_top {
```

Comment:
Don't know how it got in but if you see uncommon style of css classes like the underscore in `.fade_top`, feel free to correct it to `.fade-top` or alike